collection-scripts/gather_version

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# load shared functions and data when not sourced
+if [[ -z "$DIR_NAME" ]]; then
+    CALLED=1
+    DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    source "${DIR_NAME}/common.sh"
+fi
+
+function ocp_version() {
+    echo "openstack-k8s-operators/openstack-must-gather"> "${BASE_COLLECTION_PATH}/version"
+    ocp_version="0.0.0-unknown"
+    if [[ -n ${OS_GIT_VERSION} ]]; then
+        ocp_version="${OS_GIT_VERSION}"
+    else
+        oc_version=$(oc version -o json | jq -r .openshiftVersion)
+        if [[ -n $oc_version ]] ; then
+            ocp_version="${oc_version}"
+        fi
+    fi
+    echo ${ocp_version} >> "${BASE_COLLECTION_PATH}/version"
+}
+
+function log_version() {
+    ocp_version >> "${BASE_COLLECTION_PATH}/version"
+}
+
+if [[ $CALLED -eq 1 ]]; then
+    ocp_version
```

Comment:
should we have here `log_version` instead of `ocp_version` ?

Reply:
changed